src/interface/report/Results/Timeline/EnemyCasts.tsx

```diff
@@ -119,50 +118,48 @@ const EnemyCasts = ({
   events,
   reportCode,
   actorId,
-  showInstantCasts,
-  showChanneledCasts,
-  showInterruptedAbilities,
-  showBossAbilities,
   ...others
 }: Props) => {
-  const renderCast = (event: NpcCastEvent | NpcBeginCastEvent) => {
-    const className = '';
+  console.log('is enemy casts rerendering');
```

Comment:
as a heads up: the react devtools plugin can show you when something re-renders and explain why it re-rendered. very useful tool!